app/assets/stylesheets/darkswarm/_shop-popovers.css.scss

```diff
@@ -5,18 +5,17 @@
 // Foundation overrides
 .joyride-tip-guide.price_breakdown {
   // JS needs to be tweaked to adjust for left alignment - this is dynamic can't rewrite in CSS
-  background-color: #999;
-  color: #1f1f1f;
   margin-left: -8px;
 
-  @include box-shadow(0 1px 2px 0 rgba(0, 0, 0, 0.7));
+  @include box-shadow(0 2px 8px 0 rgba(0, 0, 0, 0.35));
 
   .joyride-content-wrapper {
     padding: 1.125rem 1.25rem 1.5rem;
     padding: 1rem;
     margin: 1%;
     width: 98%;
-    background-color: white;
+    background-color: $grey-800;
+    color: white;
```

Comment:
This is the same I spotted in https://github.com/openfoodfoundation/openfoodnetwork/pull/5263/files#r420699417.

Reply:
Okay, maybe that needs another pull request for all the places where we use `white`. But who is to say if we want to change them all together or only some of them? `$white` doesn't have a semantic meaning. I would leave it until we actually need to change it and have meaning for it.